jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdeBsmPayload.java

```diff
@@ -1,45 +1,47 @@
 /*******************************************************************************
  * Copyright 2018 572682
  * 
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License.  You may obtain a copy
- * of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except

```

Comment:
Updating classes that inherit OdeMsgPayload to now explicitly provide the OdeObject class as the type. This used to be the default, but had to change to allow for using ASN POJOs as part of the payload. Other check style changes were also applied.